.github/workflows/example-repository-dispatch.yml

```
@@ -0,0 +1,194 @@
+      - name: Install dependencies
+        run: bun install
+
+      - name: Run tests
+        run: bun test
+
+      - name: Build application
+        run: bun run build
+
+      - name: Determine dispatch strategy
+        id: check
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:34: Double quote to prevent globbing and word splitting

(shellcheck)